ezyrb/gui.py

```diff
@@ -37,14 +41,14 @@ class Gui(object):
 	:cvar string new_mu: new parameter values. The values must be separated by a comma.
 	:cvar string outfilename: name of the new output file.
 	:cvar string finish_label: string that says when the online step is done.
-	:cvar Tkinter.Label label_new_mu: label where to print the new parameter value.
-	:cvar Tkinter.Label label_error: label where to print the maximum error on the tesselation.
-	:cvar Tkinter.Label label_tria: label where to print the triangulation file path.
+	:cvar tkinter.Label label_new_mu: label where to print the new parameter value.
+	:cvar tkinter.Label label_error: label where to print the maximum error on the tesselation.
+	:cvar tkinter.Label label_tria: label where to print the triangulation file path.
 	
-	:cvar Tkinter.Label label_basis: label where to print the pod basis file path.
+	:cvar tkinter.Label label_basis: label where to print the pod basis file path.
 	
-	:cvar Tkinter.Label label_parsing_file: label where to print the parsing file path.
-	:cvar Tkinter.Label label_finish_online: label where to print the finish message.
+	:cvar tkinter.Label label_parsing_file: label where to print the parsing file path.
+	:cvar tkinter.Label label_finish_online: label where to print the finish message.
```

Comment:
Just a curiosity: does this change allow you to make the tkinter.Label linked to the tkinter documentation in the PyGeM documentation?